frontend/src/app/modify-exercise/modify-exercise.component.ts

```diff
@@ -25,6 +25,17 @@ export class ModifyExerciseComponent {
 
   difficultyRange = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
 
+  isValid(exercise: Exercise): boolean {
```

Comment:
**suggestion:** drop the exercise parameter here as it is always available as `this.exercise`